app/experimenter/experiments/models/nimbus.py

```diff
@@ -298,6 +298,11 @@ def should_end(self):
         if self.proposed_end_date:
             return datetime.date.today() >= self.proposed_end_date
 
+    @property
+    def should_end_enrollment(self):
+        if self.proposed_enrollment_end_date:
+            return datetime.date.today() >= self.proposed_enrollment_end_date
+
```

Comment:
LOL I realized right after I landed the other pr that took this method out that we'd need it back to send the email XD